Makefile

```diff
@@ -873,24 +882,22 @@ deploy/service: deploy/envoy deploy/route
 		-p SSO_PROVIDER_TYPE=${SSO_PROVIDER_TYPE} \
 		-p REGISTERED_USERS_PER_ORGANISATION=${REGISTERED_USERS_PER_ORGANISATION} \
 		-p DYNAMIC_SCALING_CONFIG=${DYNAMIC_SCALING_CONFIG} \
-		| oc apply -f - -n $(NAMESPACE)
+		| $(OC) apply -f - -n $(NAMESPACE)
 .PHONY: deploy/service
 
-
-
 # remove service deployments from an OpenShift cluster
 undeploy: IMAGE_REGISTRY ?= $(internal_image_registry)
 undeploy: IMAGE_REPOSITORY ?= $(image_repository)
 undeploy:
-	@-oc process -f ./templates/observatorium-token-refresher.yml | oc delete -f - -n $(NAMESPACE)
-	@-oc process -f ./templates/db-template.yml | oc delete -f - -n $(NAMESPACE)
-	@-oc process -f ./templates/secrets-template.yml | oc delete -f - -n $(NAMESPACE)
-	@-oc process -f ./templates/route-template.yml | oc delete -f - -n $(NAMESPACE)
-	@-oc delete -f ./templates/envoy-config-configmap.yml -n $(NAMESPACE)
-	@-oc process -f ./templates/service-template.yml \
+	@-$(OC) process -f ./templates/observatorium-token-refresher.yml | oc delete -f - -n $(NAMESPACE)
+	@-$(OC) process -f ./templates/db-template.yml | oc delete -f - -n $(NAMESPACE)
+	@-$(OC) process -f ./templates/secrets-template.yml | oc delete -f - -n $(NAMESPACE)
+	@-$(OC) process -f ./templates/route-template.yml | oc delete -f - -n $(NAMESPACE)
+	@-$(OC) delete -f ./templates/envoy-config-configmap.yml -n $(NAMESPACE)
+	@-$(OC) process -f ./templates/service-template.yml \
```

Comment:
Hey, good idea :)
Just wondering if we should be adding the references in the later half also? Similar to this:

```diff
-        @-$(OC) process -f ./templates/observatorium-token-refresher.yml | oc delete -f - -n $(NAMESPACE)
+        @-$(OC) process -f ./templates/observatorium-token-refresher.yml | $(OC) delete -f - -n $(NAMESPACE)
```

Reply:
That's correct. I missed those. I will update the contents :+1:

Reply:
updated